src/hooks/operators.ts

```diff
@@ -91,6 +95,14 @@ export function useOperatorByIdQuery(operatorId = '') {
     })
 }
 
+export function invalidateActiveOperatorByIdQueries(operatorId: string) {
+    getQueryClient().invalidateQueries({
+        queryKey: operatorByIdQueryKey(operatorId),
+        exact: true,
+        refetchType: 'active',
+    })
+}
+
```

Comment:
This will invalidate all operator queries and only refetch the actve ones. Lovely.